backends/nxp/aten_passes/neutron_aten_pass_manager.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2025 NXP
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Callable
+
+import torch
+
+from executorch.backends.nxp.aten_passes.fuse_batch_norm_with_conv_pass import (
```

Comment:
nit: can become unwieldly since there are a few other permutations of "batch norm" fusion, maybe just one file for fuse_batch_norm would be good.

Reply:
Agree with you. Maybe we can create subdirectories in `backend/transforms` based on what dialect/ir is pass targeting?